src/rev_ai/models/asynchronous/summarization_options.py

```diff
@@ -0,0 +1,63 @@
+from .summarization_formatting_options import SummarizationFormattingOptions
+from .summarization_job_status import SummarizationJobStatus
+from ..nlp_model import NlpModel
+
+"""Summarization request options."""
+
+
+class SummarizationOptions:
+    def __init__(
+            self,
+            prompt: str = None,
+            model: NlpModel = None,
+            formattingType: SummarizationFormattingOptions = None
+    ):
```

Comment:
Is this formatting correct?

Reply:
I used auto formatter as suggested with linter, no complaints.

Reply:
I follow PEP8, which is commonly accepted standard for python formatting, PyCharm supports PEP 8 and suggests this formatting here. I Used autoformat everywhere where reasonable.